include/crm/common/output.h

```diff
@@ -698,7 +698,8 @@ pcmk__output_create_html_node(pcmk__output_t *out, const char *element_name, con
  * \param[in]     ...    A NULL-terminated key/value list of attributes.
  */
 void
-pcmk__html_add_header(xmlNodePtr parent, const char *name, ...);
+pcmk__html_add_header(xmlNodePtr parent, const char *name, ...)
+__attribute__((sentinel));
```

Comment:
Since we support non-gcc compilers, we conditionally define macros for gcc-specific syntax (see e.g. the bottom of config.h.in).

Although searching for __attribute__, I see we haven't been consistent. We use quite a few directly. I guess those are at least supported by clang since we use that in CI.

Reply:
Yeah, we are kind of all over the place with attributes.  I added __attribute__((unused)) in one spot a little while back, but we have _GL_UNUSED defined so I could use that instead.  Looks like clang doesn't support sentinel (or perhaps calls it something else) so I'll add the conditional definition for it.